diff --git a/.fabro/scratch/01KRJKXWDAMEG4SYJ72YYK839H/parallel/spec_fanout/spec_claude b/.fabro/scratch/01KRJKXWDAMEG4SYJ72YYK839H/parallel/spec_fanout/spec_claude
index 7d2f8b0..18ae938 160000
--- a/.fabro/scratch/01KRJKXWDAMEG4SYJ72YYK839H/parallel/spec_fanout/spec_claude
+++ b/.fabro/scratch/01KRJKXWDAMEG4SYJ72YYK839H/parallel/spec_fanout/spec_claude
@@ -1 +1 @@
-Subproject commit 7d2f8b02c5e38b7e9ff7e4dd859f20d61570a908
+Subproject commit 18ae93851c148b8ca6cbb12e495679031810bcf8
diff --git a/.fabro/scratch/01KRJKXWDAMEG4SYJ72YYK839H/parallel/spec_fanout/spec_deepseek b/.fabro/scratch/01KRJKXWDAMEG4SYJ72YYK839H/parallel/spec_fanout/spec_deepseek
index 7d2f8b0..18ae938 160000
--- a/.fabro/scratch/01KRJKXWDAMEG4SYJ72YYK839H/parallel/spec_fanout/spec_deepseek
+++ b/.fabro/scratch/01KRJKXWDAMEG4SYJ72YYK839H/parallel/spec_fanout/spec_deepseek
@@ -1 +1 @@
-Subproject commit 7d2f8b02c5e38b7e9ff7e4dd859f20d61570a908
+Subproject commit 18ae93851c148b8ca6cbb12e495679031810bcf8
diff --git a/.fabro/scratch/01KRJKXWDAMEG4SYJ72YYK839H/parallel/spec_fanout/spec_kimi b/.fabro/scratch/01KRJKXWDAMEG4SYJ72YYK839H/parallel/spec_fanout/spec_kimi
index 7d2f8b0..18ae938 160000
--- a/.fabro/scratch/01KRJKXWDAMEG4SYJ72YYK839H/parallel/spec_fanout/spec_kimi
+++ b/.fabro/scratch/01KRJKXWDAMEG4SYJ72YYK839H/parallel/spec_fanout/spec_kimi
@@ -1 +1 @@
-Subproject commit 7d2f8b02c5e38b7e9ff7e4dd859f20d61570a908
+Subproject commit 18ae93851c148b8ca6cbb12e495679031810bcf8
