diff --git a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/app_store_research b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/app_store_research
index 3ee1d86..2b1d5e2 160000
--- a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/app_store_research
+++ b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/app_store_research
@@ -1 +1 @@
-Subproject commit 3ee1d8678ab53e394f0978603d79f7af61b29566
+Subproject commit 2b1d5e2514657ee9e37de617e987734190da8ae5
diff --git a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/competitor_research b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/competitor_research
index 3ee1d86..2b1d5e2 160000
--- a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/competitor_research
+++ b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/competitor_research
@@ -1 +1 @@
-Subproject commit 3ee1d8678ab53e394f0978603d79f7af61b29566
+Subproject commit 2b1d5e2514657ee9e37de617e987734190da8ae5
diff --git a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/design_pattern_research b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/design_pattern_research
index 3ee1d86..2b1d5e2 160000
--- a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/design_pattern_research
+++ b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/design_pattern_research
@@ -1 +1 @@
-Subproject commit 3ee1d8678ab53e394f0978603d79f7af61b29566
+Subproject commit 2b1d5e2514657ee9e37de617e987734190da8ae5
diff --git a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/reddit_research b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/reddit_research
index 3ee1d86..2b1d5e2 160000
--- a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/reddit_research
+++ b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/reddit_research
@@ -1 +1 @@
-Subproject commit 3ee1d8678ab53e394f0978603d79f7af61b29566
+Subproject commit 2b1d5e2514657ee9e37de617e987734190da8ae5
