diff --git a/.fabro/scratch/01KRGE9QQP3MQBYCR6WZPDSCXH/parallel/review_fanout/review_deepseek b/.fabro/scratch/01KRGE9QQP3MQBYCR6WZPDSCXH/parallel/review_fanout/review_deepseek
new file mode 160000
index 0000000..69dd07e
--- /dev/null
+++ b/.fabro/scratch/01KRGE9QQP3MQBYCR6WZPDSCXH/parallel/review_fanout/review_deepseek
@@ -0,0 +1 @@
+Subproject commit 69dd07e0a393253537773ed19e4aea5749cef2ca
diff --git a/.fabro/scratch/01KRGE9QQP3MQBYCR6WZPDSCXH/parallel/review_fanout/review_kimi b/.fabro/scratch/01KRGE9QQP3MQBYCR6WZPDSCXH/parallel/review_fanout/review_kimi
new file mode 160000
index 0000000..69dd07e
--- /dev/null
+++ b/.fabro/scratch/01KRGE9QQP3MQBYCR6WZPDSCXH/parallel/review_fanout/review_kimi
@@ -0,0 +1 @@
+Subproject commit 69dd07e0a393253537773ed19e4aea5749cef2ca
diff --git a/.fabro/scratch/01KRGE9QQP3MQBYCR6WZPDSCXH/parallel/review_fanout/review_qwen b/.fabro/scratch/01KRGE9QQP3MQBYCR6WZPDSCXH/parallel/review_fanout/review_qwen
new file mode 160000
index 0000000..69dd07e
--- /dev/null
+++ b/.fabro/scratch/01KRGE9QQP3MQBYCR6WZPDSCXH/parallel/review_fanout/review_qwen
@@ -0,0 +1 @@
+Subproject commit 69dd07e0a393253537773ed19e4aea5749cef2ca
