diff --git a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/app_store_research b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/app_store_research
index c71cf49..3ee1d86 160000
--- a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/app_store_research
+++ b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/app_store_research
@@ -1 +1 @@
-Subproject commit c71cf49cb7de0bcc77145c3c2ad7322ed529e422
+Subproject commit 3ee1d8678ab53e394f0978603d79f7af61b29566
diff --git a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/competitor_research b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/competitor_research
index c71cf49..3ee1d86 160000
--- a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/competitor_research
+++ b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/competitor_research
@@ -1 +1 @@
-Subproject commit c71cf49cb7de0bcc77145c3c2ad7322ed529e422
+Subproject commit 3ee1d8678ab53e394f0978603d79f7af61b29566
diff --git a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/design_pattern_research b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/design_pattern_research
index c71cf49..3ee1d86 160000
--- a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/design_pattern_research
+++ b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/design_pattern_research
@@ -1 +1 @@
-Subproject commit c71cf49cb7de0bcc77145c3c2ad7322ed529e422
+Subproject commit 3ee1d8678ab53e394f0978603d79f7af61b29566
diff --git a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/reddit_research b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/reddit_research
index c71cf49..3ee1d86 160000
--- a/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/reddit_research
+++ b/.fabro/scratch/01KRZE8H8QYQ8WDDSG0TNW84S9/parallel/research_fanout/reddit_research
@@ -1 +1 @@
-Subproject commit c71cf49cb7de0bcc77145c3c2ad7322ed529e422
+Subproject commit 3ee1d8678ab53e394f0978603d79f7af61b29566
