diff --git a/.fabro/scratch/01KRZAG54XZQN1R4GA403G97KD/parallel/research_fanout/app_store_research b/.fabro/scratch/01KRZAG54XZQN1R4GA403G97KD/parallel/research_fanout/app_store_research
new file mode 160000
index 0000000..e1b5602
--- /dev/null
+++ b/.fabro/scratch/01KRZAG54XZQN1R4GA403G97KD/parallel/research_fanout/app_store_research
@@ -0,0 +1 @@
+Subproject commit e1b560294200c9a346759a0b069135878adee2fe
diff --git a/.fabro/scratch/01KRZAG54XZQN1R4GA403G97KD/parallel/research_fanout/competitor_research b/.fabro/scratch/01KRZAG54XZQN1R4GA403G97KD/parallel/research_fanout/competitor_research
new file mode 160000
index 0000000..e1b5602
--- /dev/null
+++ b/.fabro/scratch/01KRZAG54XZQN1R4GA403G97KD/parallel/research_fanout/competitor_research
@@ -0,0 +1 @@
+Subproject commit e1b560294200c9a346759a0b069135878adee2fe
diff --git a/.fabro/scratch/01KRZAG54XZQN1R4GA403G97KD/parallel/research_fanout/design_pattern_research b/.fabro/scratch/01KRZAG54XZQN1R4GA403G97KD/parallel/research_fanout/design_pattern_research
new file mode 160000
index 0000000..e1b5602
--- /dev/null
+++ b/.fabro/scratch/01KRZAG54XZQN1R4GA403G97KD/parallel/research_fanout/design_pattern_research
@@ -0,0 +1 @@
+Subproject commit e1b560294200c9a346759a0b069135878adee2fe
diff --git a/.fabro/scratch/01KRZAG54XZQN1R4GA403G97KD/parallel/research_fanout/reddit_research b/.fabro/scratch/01KRZAG54XZQN1R4GA403G97KD/parallel/research_fanout/reddit_research
new file mode 160000
index 0000000..e1b5602
--- /dev/null
+++ b/.fabro/scratch/01KRZAG54XZQN1R4GA403G97KD/parallel/research_fanout/reddit_research
@@ -0,0 +1 @@
+Subproject commit e1b560294200c9a346759a0b069135878adee2fe
