diff --git a/.fabro/scratch/01KRZAQ1RHATXXVRH63RBP7HE2/parallel/research_fanout/app_store_research b/.fabro/scratch/01KRZAQ1RHATXXVRH63RBP7HE2/parallel/research_fanout/app_store_research
new file mode 160000
index 0000000..2081058
--- /dev/null
+++ b/.fabro/scratch/01KRZAQ1RHATXXVRH63RBP7HE2/parallel/research_fanout/app_store_research
@@ -0,0 +1 @@
+Subproject commit 2081058c44a2780c18907aad963365e893456416
diff --git a/.fabro/scratch/01KRZAQ1RHATXXVRH63RBP7HE2/parallel/research_fanout/competitor_research b/.fabro/scratch/01KRZAQ1RHATXXVRH63RBP7HE2/parallel/research_fanout/competitor_research
new file mode 160000
index 0000000..2081058
--- /dev/null
+++ b/.fabro/scratch/01KRZAQ1RHATXXVRH63RBP7HE2/parallel/research_fanout/competitor_research
@@ -0,0 +1 @@
+Subproject commit 2081058c44a2780c18907aad963365e893456416
diff --git a/.fabro/scratch/01KRZAQ1RHATXXVRH63RBP7HE2/parallel/research_fanout/design_pattern_research b/.fabro/scratch/01KRZAQ1RHATXXVRH63RBP7HE2/parallel/research_fanout/design_pattern_research
new file mode 160000
index 0000000..2081058
--- /dev/null
+++ b/.fabro/scratch/01KRZAQ1RHATXXVRH63RBP7HE2/parallel/research_fanout/design_pattern_research
@@ -0,0 +1 @@
+Subproject commit 2081058c44a2780c18907aad963365e893456416
diff --git a/.fabro/scratch/01KRZAQ1RHATXXVRH63RBP7HE2/parallel/research_fanout/reddit_research b/.fabro/scratch/01KRZAQ1RHATXXVRH63RBP7HE2/parallel/research_fanout/reddit_research
new file mode 160000
index 0000000..2081058
--- /dev/null
+++ b/.fabro/scratch/01KRZAQ1RHATXXVRH63RBP7HE2/parallel/research_fanout/reddit_research
@@ -0,0 +1 @@
+Subproject commit 2081058c44a2780c18907aad963365e893456416
